spec/requests/objects_controller_checksums_spec.rb

```diff
@@ -201,15 +201,17 @@
         it 'body has additional information from the exception if available' do
           post checksums_objects_url, params: { druids: [], format: :json }, headers: valid_auth_header
           expect(response).to have_http_status(:bad_request)
-          expect(response.body).to eq '400 Bad Request: Identifier has invalid suri syntax:  nil or empty'
+          error_response = JSON.parse(response.body)['errors'].first
+          expect(error_response['detail']).to include('does not match value: , example: druid:bc123df4567')
```

Comment:
we no longer return the error codes?  I think we put them in there so the consumer, via preservation-client, could see them?  Or am I garbling things in my mind?   Are the errors on the consumer side too repetitious as it is???

Reply:
No, because this is a committee error code. The client should still get the status; it just won't be duplicated in the body/message.